frontend/src/features/fields/fieldService.tsx

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+
+const API_URL = '/api/fields/';
+const SERVER_URL = process.env.REACT_APP_SERVER_URL || '';
```

Comment:
The server url is meant to go before the api url, so you can either define api like 
`const API_URL = '${SERVER_URL}/api/fields/';` or alternatively do `SERVER_URL + API_URL` at each call